app/controllers/api/v2/hosts_controller.rb

```diff
@@ -75,6 +75,13 @@ def show
             param_group :interface_attributes, ::Api::V2::InterfacesController
           end
           param :compute_attributes, Hash, :desc => N_("Additional compute resource specific attributes.")
+
+          Facets.registered_facets.values.each do |facet_config|
+            next unless facet_config.api_param_group && facet_config.api_controller
+            param "#{facet_config.name}_attributes".to_sym, Hash, :desc => facet_config.api_param_group_description || N_("Parameters for host's %s facet" % facet_config.name) do
```

Comment:
Translation syntax should be `(N_("Parameters for host's %s facet") % facet_config.name)`